### ISSUE TITLE
서비스 체험 페이지 만들기

### ISSUE THREAD

**Issue by hwna00** [osamhack2021/AI_WEB_AICARE_AIM_2021#7]
## 목적

- 서비스에 가입하지 않은 사용자들에게 체험할 수 있는 페이지 제작.

## 상세내용

- 저희 서비스가 웹이다보니 사용자가 검색으로 들어올텐데 그 사용자들에게 바로 회원가입, 로그인을 한 후에 기능들을 사용해볼 수 있게 하는 것보다 먼저 체험을 한 후 가입하게 하는 것이 더 좋을 것 같습니다.

## 참고사항

-

**Comment by DINFBO** [comment]
네 제 생각도 비슷합니다. 1회성으로 테스트를 하고, 좀더 세밀한 분석 혹은 테스트 데이터의 저장은 로그인한 유저에게만 제공해서 가입을 유도하는게 좋을거같습니다
